### PR TITLE
add volume buttons to control background music 

### PR DESCRIPTION
**Pull request by menynexi**
volume Icons have been added to  gameStage to give a more retro game ascetic 

Screenshot 2020-07-27 at 7 00 16 AM

Screenshot 2020-07-27 at 7 00 03 AM

feel free to look at the live demo : https://cs-career-step-2020.wm.r.appspot.com/

### CODE REVIEW

**Comment by menynexi** [comment]
TODO: revise gameStage and align everything accordingly

**Comment by menynexi** [comment]
This will be committed in my next PR

File: src/main/webapp/gameDialogue.js [hunk 1]
```diff
@@ -13,13 +13,19 @@
 // limitations under the License.
 const FINAL_STAGE_BUTTON_VALUE = 'COMPLETE PATH';
 const FINAL_STAGE_REDIRECTION = 'CompletedPath.html';
+const MUSIC_CONTAINER = 'musicicon';
```
Comment (lattai) [inline src/main/webapp/gameDialogue.js:16]: musicIcon

File: src/main/webapp/gameStage.css [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+    position: absolute;
+    margin-left: 50%;
+    margin-right: 50%;
+    margin-top: 25%;
+}
+
+audio {
+float: left;
+}
+
```
Comment (lattai) [inline src/main/webapp/gameStage.css:458]: It seems pretty Identical to style.css except for the last 10 lines. If they're going to be two separate files, there should probably be a greater difference between the two 450 line files or just the one style sheet

File: src/main/webapp/gameDialogue.js [hunk 3]
```diff
@@ -13,13 +13,19 @@
...
+const UNMUTE_ICON = '<img src="icons/unmute.gif" alt="successfull promotion meme">';
```
Comment (WillHipschman) [inline src/main/webapp/gameDialogue.js:17]: alt text is wrong for both of these

File: src/main/webapp/gameDialogue.js [hunk 4]
```diff
@@ -95,12 +101,14 @@ function modifyIfFinalStage() {
...
 function playmusic() {
-  //TODO: change the play and pause to toggle on and off icons
-  if(document.getElementById('player-button').innerText == "PAUSE"){
-    document.getElementById('player-button').innerText = "PLAY";
-  }
-  else{
-    document.getElementById('player-button').innerText = "PAUSE";
+  if(isPlayingmusic) {
+    const promotionMEME = document.getElementById(MUSIC_CONTAINER);
+    promotionMEME.innerHTML = UNMUTE_ICON;
```
Comment (WillHipschman) [inline src/main/webapp/gameDialogue.js:106]: why is this named promotionMEME?

File: src/main/webapp/gameStage.css [hunk 5]
```diff
@@ -0,0 +1,458 @@
...
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+}
+
+.step-team-button:hover {
+  background-color: #555; 
+}
+
+/** Responsive media classes to addapt to diffrent devices **/
```
Comment (WillHipschman) [inline src/main/webapp/gameStage.css:279]: typo addapt-->adapt

File: src/main/webapp/gameStage.css (hunk 2 above)
Comment (WillHipschman) [reply src/main/webapp/gameStage.css:458]: +1

File: src/main/webapp/gameDialogue.js (hunk 1 above)
Comment (DirtGrubDylan) [reply src/main/webapp/gameDialogue.js:16]: `const MUSIC_ICON`

File: src/main/webapp/gameDialogue.js (hunk 3 above)
Comment (DirtGrubDylan) [reply src/main/webapp/gameDialogue.js:17]: +1

File: src/main/webapp/gameDialogue.js (hunk 4 above)
Comment (DirtGrubDylan) [reply src/main/webapp/gameDialogue.js:106]: +1

Also, it looks like `const promotionMeme = document.getElementById(MUSIC_CONTAINER)` could be set outside of the `if/else` block

File: src/main/webapp/gameDialogue.js (hunk 1 above)
Comment (menynexi) [reply src/main/webapp/gameDialogue.js]: thank you for the naming suggestion

File: src/main/webapp/gameDialogue.js (hunk 3 above)
Comment (menynexi) [reply src/main/webapp/gameDialogue.js]: nice catch thank you, fixed

File: src/main/webapp/gameStage.css (hunk 5 above)
Comment (menynexi) [reply src/main/webapp/gameStage.css]: nice catch I had not cough tit in the original style guide

File: src/main/webapp/gameDialogue.js [hunk 6]
```diff
@@ -13,8 +13,13 @@
 // limitations under the License.
 const FINAL_STAGE_BUTTON_VALUE = 'COMPLETE PATH';
 const FINAL_STAGE_REDIRECTION = 'CompletedPath.html';
+const MUSIC_ICON= 'musicIcon';
+const UNMUTE_ICON = '<img src="icons/unmute.gif" alt="unmuted icon">';
+const MUTE_ICON = '<img src="icons/mute.png" alt="muted icon">';
 
 var dialogueArray;
+var i;
```
Comment (WillHipschman) [inline src/main/webapp/gameDialogue.js:21]: What is variable i? It looks unused...

Comment (menynexi) [reply src/main/webapp/gameDialogue.js:21]: it is an iterator for dialogue

Comment (DirtGrubDylan) [reply src/main/webapp/gameDialogue.js:21]: rename to something more descriptive `dialogueIterator`

Comment (menynexi) [reply src/main/webapp/gameDialogue.js]: i was a merging conflict on my par the var i was replaced in the latest commit to master

**Pull request merged**